content/en/agent/basic_agent_usage/windows.md

```diff
@@ -23,26 +23,34 @@ This page outlines the basic features of the Windows Datadog Agent. If you haven
-| `HOSTNAME`        | String | Configures the hostname reported by the Agent to Datadog (overrides any hostname calculated at runtime).                                                                                          |
-| `LOGS_ENABLED`    | String | Enable (`"true"`) or disable (`"false"`) the log collection feature in the configuration file. Logs are disabled by default.                                                                      |
-| `APM_ENABLED`     | String | Enable (`"true"`) or disable (`"false"`) the APM Agent in the configuration file. APM is enabled by default.                                                                                      |
-| `PROCESS_ENABLED` | String | Enable (`"true"`) or disable (`"false"`) the Process Agent in the configuration file. The Process Agent is disabled by default.                                                                   |
-| `CMD_PORT`        | Number | A valid port number between 0 and 65534. The Datadog Agent uses port 5001 by default for it's control API. If that port is already in use by another program, the default may be overridden here. |
-| `PROXY_HOST`      | String | If using a proxy, sets your proxy host. [Learn more about using a proxy with the Datadog Agent][3].                                                                                               |
-| `PROXY_PORT`      | Number | If using a proxy, sets your proxy port. [Learn more about using a proxy with the Datadog Agent][3].                                                                                               |
-| `PROXY_USER`      | String | If using a proxy, sets your proxy user. [Learn more about using a proxy with the Datadog Agent][3].                                                                                               |
-| `PROXY_PASSWORD`  | String | If using a proxy, sets your proxy password. [Learn more about using a proxy with the Datadog Agent][3].                                                                                           |
+To install the Agent, you may simply download and run the Datadog Agent MSI as Administrator.
+
+Optionally, install the Agent with the command line to customize the install settings and/or run an unattended install. Each configuration item is added as an install property to the command line. For instance, the following command installs the Agent and configures the Agent configuration file with the `<DATADOG_API_KEY>`.
+
+* cmd: `msiexec /i datadog-agent-6-latest.amd64.msi APIKEY="<DATADOG_API_KEY>"`
+* Powershell: `Start-Process msiexec -ArgumentList '/i datadog-agent-6-latest.amd64.msi APIKEY="<DATADOG_API_KEY>"'`
+
+**Note**: Use the `/qn /i` options instead of `/i` to run an unattended install. The unattended install may take up to a few minutes to complete.
```

Comment:
```suggestion
**Note**: Use the `/qn /i` options instead of `/i` to run an unattended install. The unattended install may take a few minutes to complete.
```

👍